st2actions/st2actions/notifier/notifier.py

```diff
@@ -159,6 +181,22 @@ def _post_notify_subsection_triggers(self, liveaction=None, execution_id=None,
             if len(failed_routes) > 0:
                 raise Exception('Failed notifications to routes: %s' % ', '.join(failed_routes))
 
+    def _build_jinja_context(self, liveaction, execution):
+        context = {SYSTEM_KV_PREFIX: KeyValueLookup()}
+        context.update(liveaction.parameters)
+        context.update({ACTION_CONTEXT_KV_PREFIX: liveaction.context})
```

Comment:
Hmm, the content of this value is opaque. I suspect it also depends on the source of the action etc.